airbyte-server/src/main/java/io/airbyte/server/handlers/ConnectionsHandler.java

```diff
@@ -162,7 +159,11 @@ public ConnectionRead createConnection(final ConnectionCreate connectionCreate)
 
     trackNewConnection(standardSync);
 
-    return buildConnectionRead(connectionId);
+    if (featureFlags.usesNewScheduler()) {
+      temporalWorkerRunFactory.createNewSchedulerWorkflow(connectionId);
```

Comment:
are there any side effects if we write a standard sync and fail to create a new scheduler workflow for that connection?

Reply:
yes, the sync won't be schedule and we won't be able to signal if since all the signal expect it to be running. We rely on the temporal lib to make sure that the workflow is properly created.

Reply:
okay so we don't need to wrap this in a transaction to roll back the standard sync creation or anything?

Reply:
Done, I remove the standard sync before failing again.